generator/src/test/java/com/scottlogic/deg/generator/cucumber/EqualTo.feature

```diff
@@ -760,10 +724,7 @@ Scenario: Not 'equalTo' a number and 'ofType' string is successful
 
 Scenario: Not 'equalTo' a date and 'ofType' string is successful
```

Comment:
is the name of this scenario historic?
does it still check a `typeOf` constraint?